packages/atomic/src/components/result-template-components/atomic-result-date-value/atomic-result-date-value.tsx

```diff
@@ -0,0 +1,42 @@
+import {Component, Prop, Element} from '@stencil/core';
+import {Result, ResultTemplatesHelpers} from '@coveo/headless';
+import dayjs from 'dayjs';
+import {ResultContext} from '../result-template-decorators';
+
+/**
+ * The ResulDateValue component renders the value of a date result property.
+ */
+@Component({
+  tag: 'atomic-result-date-value',
```

Comment:
Could we shorten the name of the component to `atomic-result-date`? I don't think the `value` at the end is adding _value_ to the name. Suggestion:

- `atomic-result-price`
- `atomic-result-date`
- `atomic-result-number`
- `atomic-result-text`

What do you think?